NEON_plant_phenology.Rmd

```diff
@@ -231,6 +234,138 @@ then disappear again in October. But there might be differences between species
 in a species. One way we could try to assess this is to fit the same model to subgroups of the data and
 then compare the fitted parameters to see if there are differences. 
 
+### Degree Days
+
+
+Get the max and min temperatures to calculate Growing Degree Days using the Average Method.
+```{r}
+temp$date <- as.Date(temp$startDateTime)
+
+temp$tempSingleMean <- as.numeric(temp$tempSingleMean)
```

Comment:
Not strictly necessary at all, but it's possible to tidyverse-ify this like so:

```R
temp <- temp %>%
    mutate(date = as.Date(startDateTime),
           tempSingleMean = as.numeric(tempSingleMean))
```

Might help with consistency - although it could be equally useful for students to know of this alternate method. Your call!